page.codeberg.petsoi.wordle.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "page.codeberg.petsoi.wordle",
```

Comment:
the name itself might be a trademark infringement.

Reply:
please change the name.

Reply:
You are totally right guys. I changed it Words!